lib/core/src/lib/card-view/components/card-view-selectitem/card-view-selectitem.component.spec.ts

```diff
@@ -73,14 +73,6 @@ describe('CardViewSelectItemComponent', () => {
     });
 
     describe('Rendering', () => {
-        it('should render the label', () => {
```

Comment:
should this test be removed? Label is still rendered, just in different place so we can still check if that is rendered: 
<img width="838" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/168af304-f5f7-4b00-9ea5-fd3b8c8b3c1a">

Reply:
I only removed this test since after the change, we basically went from our custom implementation of showing a label for a form field, to angular material's proposed way of showing labels for form fields. Since it was something coming from angular material, I figured there is no need to have a unit test for it anymore, since it would already be covered by them. 
However, the label code might change after @MichalKinas 's review comment. If that is indeed the case then I will restore this unit test

Reply:
@swapnil-verma-gl Fine for me, you can skip this comment. Only thing which I thought what is not tested by "Angular material" is that they can test only if label is displayed but they are not testing that this is coming from "property.label" so if anyone will change this part: 

"{{ property.label }}" 

then it won't be protected. So somebody can use different field here or something. But this is small thing so if you think it is not needed then I'm fine to skip it. 


I will approve that PR when you resolve this comment from ACA https://github.com/Alfresco/alfresco-content-app/pull/3505#issuecomment-1820596560 as there is still some issue in ACA even with these ADF changes @swapnil-verma-gl

Reply:
Thank you @AleksanderSklorz. The alignment issue has been resolved with [this](https://github.com/Alfresco/alfresco-ng2-components/pull/9101/commits/9b8001578fb3f4876fb38c14aaf5dfe941b51184) commit

Reply:
I've re-added this unit test, and added a another unit test corresponding to the presence of mat-label as well, as part of resolving @MichalKinas 's review comment